ActiveLabeler-main/Self-Supervised-Learner/dali_utils/dali_transforms.py

```diff
@@ -0,0 +1,93 @@
+import nvidia.dali.ops as ops
+import nvidia.dali.types as types
+from nvidia.dali.pipeline import Pipeline
+from torchvision.datasets import ImageFolder
+
+
+class SimCLRTransform(Pipeline):
+    def __init__(
+        self,
+        DATA_PATH,
```

Comment:
All caps should ideally only be used for constants.

References:
[PEP8](https://www.python.org/dev/peps/pep-0008/#constants)
[Python naming convention](https://github.com/naming-convention/naming-convention-guides/tree/master/python)